src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/calculator/MovePrice.kt

```diff
@@ -19,4 +43,18 @@ data class MovePrice(
 
 data class JourneyPrice(
         val journeyWithEvents: JourneyWithEvents,
-        val priceInPence: Int?)
\ No newline at end of file
+        val priceInPence: Int?)
+
+
+fun List<MovePrices>.summary(): PriceSummary {
+    return PriceSummary(
+            sumByDouble { it.summary.percentage },
+            sumBy { it.summary.volume },
+            sumBy { it.summary.volumeUnpriced },
+            sumBy { it.summary.totalPriceInPence }
+    )
+}
+
+fun List<MovePrices>.withType(movePriceType: MovePriceType) =
+        find { it.movePriceType == movePriceType } ?: MovePrices(movePriceType, listOf(), PriceSummary())
```

Comment:
@matt-crowson-moj previously I returned an exception if it wasn't found but put this in to keep the generator test happy. Should probably revisit.